frontend/app/controllers/User.scala

```diff
@@ -19,8 +19,6 @@ trait User extends Controller {
     Ok(json).withCookies(Cookie("GU_MEM", GuMemCookie.encodeUserJson(json), secure = true, httpOnly = false))
   }
 
-
-
```

Comment:
Always worth trying to reduce the size of your diff, so consider leaving out formatting changes if they're not essential (can always tidy up in a separate PR)